src/main/webapp/help-initScript.html

```diff
@@ -9,7 +9,7 @@
         <p>
           <pre><code># Install Java
 sudo apt-get -y update
-sudo apt-get install -y openjdk-7-jdk
+sudo apt-get install -y openjdk-8-jdk
 # Install Git
 sudo apt-get install -y git</code></pre>
```

Comment:
shall we move the sample (powershell) below into the sample script: https://raw.githubusercontent.com/Azure/azure-devops-utils/master/powershell/Jenkins-Windows-Init-Script-no-secrets.ps1?